client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -7,8 +7,9 @@ import gql from "graphql-tag";
 import styled from "styled-components";
 import { graphql } from "react-apollo";
 import { Helmet } from "react-helmet";
+const ReactMarkdown = require("react-markdown");
```

Comment:
Could this be done in ES6 notation for consistency?
```suggestion
import * as ReactMarkdown from "react-markdown";
```

Reply:
Interesting. I originally had `import ReactMarkdown from "react-markdown"` and my editor complained the module had no default export. Looking at `node_modules` it looks like the react-markdown package uses Typescript's `export =` syntax which is not the same as `export default` syntax. Typescript documentation (https://www.typescriptlang.org/docs/handbook/modules.html) actually advises using `import ReactMarkdown = require("react-markdown")` in this case, which doesn't even look like JS to me and so I went with the `require` syntax which I knew webpack could handle. Nice to see that `import * as` syntax works though. Seems like the best option!